docs/context/ref/SearchCriteria.md

```diff
@@ -0,0 +1,60 @@
+---
+id: SearchCriteria
+sidebar_label: SearchCriteria
+title: SearchCriteria
+hide_title: true
+---
+# `SearchCriteria`
+
+A context type representing a search criteria that can be used to filter and display messages from a chat.
+
+## Type
+
+`fdc3.searchCriteria`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/searchCriteria.schema.json
+
+## Details
+
+| Property         | Type            | Required | Example Value        |
+|------------------|-----------------|----------|----------------------|
+| `type`           | string          | Yes      | `'fdc3.searchCriteria'`     |
+| `contexts`    | (Instrument&#124;<br>Contact&#124;<br>Organization&#124;<br>string)[]  | Yes      | `[{ type: "fdc3.instrument", id: { ticker: "AAPL" }}, { type: "fdc3.contact", name: "Jane Doe", id: { email: "jane.doe@mail.com" }}, { type: "fdc3.organization", name: "Symphony" }, "#OrderID45788422"]` |
```

Comment:
I'd recommend restoring the HTML entities as they make a big difference when rendered (despite looking nuts in the markdown source)

Reply:
I'm struggling with the field name as it doesn't accept arbitrary contexts, rather several specific types - one of which isn't actually a context but a string. 

Either it needs renaming to `criteria` or add optional named fields for each type (which allows setting none to reset the filter BTW). 

Do you expect more than one contact or instrument to be added? If so those separate fields would also be arrays, otherwise single, optional objects.

Do you think this might extend to additional types in future. That is more likely if the type is generic and used beyond chat. In which case `contexts` is a valid name, if the string criteria is moved to a separate field.

Reply:
I'll rename 'contexts' into 'criteria'.

@kriswest I don't get your point: '_If so those separate fields would also be arrays_'

I have in mind to use something like that in case of multiple Contact, Instrument:
```ts
criteria: [
        {
            type: "fdc3.contact",
            name: "Jane Doe",
            id: {
                email: "jane.doe@mail.com"
            }
        },
        {
            type: "fdc3.contact",
            name: "Jean Dupont",
            id: {
                email: "jean.dupond@mail.com"
            }
        },
        {
            type: "fdc3.organization",
            name: "Symphony"
        },
        {
            type: "fdc3.instrument",
            id: {
                ticker: "AAPL"
            }
        },
        {
            type: "fdc3.instrument",
            id: {
                ticker: "ABC"
            }
        }
]
```
So, there is only 1 array accepting any Contact, Intents, et .... 
Obviously, what is missing here is the AND/OR/NOT criteria ;)

Reply:
> Do you expect more than one contact or instrument to be added? If so those separate fields would also be arrays, otherwise single, optional objects.

What I meant was, if you separated out the 'contexts' into separate fields for each type, would you be accept just a single contact or an array of contacts. 

The point is moot if you go to AND/OR/NOT arrays as that clearly defines any combinations

Reply:
I have in mind to use only one array (criteria) and not one field/array per type.
This array could contain objects of different types (like Contact and Organization and Instrument...)
@kriswest Do you agree with this solution? Or would you prefer an array by type?